app/helpers/sufia/sufia_helper_behavior.rb

```diff
@@ -91,6 +91,17 @@ def link_to_field(name, value, label = nil, facet_hash = {})
       link_to(label, main_app.search_catalog_path(state))
     end
 
+    def index_facet_link options={}
```

Comment:
On the bright side, our code creates better html.  I do not know anything about microdata, but from my quick read nested itemprop does not seem to be valid. @mtribone  noticed the nested itemprops last week. The code in the PR creates
```
<span itemprop=\"creator\"><a href=\"/catalog?f%5Bcreator_sim%5D%5B%5D=Justin\">Justin</a>, <a href=\"/catalog?f%5Bcreator_sim%5D%5B%5D=Joe\">Joe</a></span>
```
The blacklight link to search creates:
```
<span itemprop=\"creator\"><a href=\"/catalog?f%5Bcreator_tesim%5D%5B%5D=Justin\"><span itemprop=\"creator\">Justin</span></a></span> and <span itemprop=\"creator\"><a href=\"/catalog?f%5Bcreator_tesim%5D%5B%5D=Joe\"><span itemprop=\"creator\">Joe</span></a></span>
```